redis_enterprise/README.md

```diff
@@ -0,0 +1,65 @@
+# Agent Check: Redis Enterprise
+
+## Overview
+
+The [Redis Enterprise][1] integration provides metrics for three critical aspects of a cluster; database, node, and shard. This integration enables fine-grained reporting that makes it easier to detect problems before they become critical. For a full list see the 'Metrics' section.
```

Comment:
Could you provide 1-2 sentences on what your product does?

Reply:
example - this is where you mention that this is on-prem etc so when you release the cloud version, customers/users can clearly tell the differences.